CHANGELOG.md

```diff
@@ -55,7 +57,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Known Issues
 
-- JWT expiry not taken into account in authorization response cache [PR #283](https://github.com/3scale/apicast/pull/283) / [Issue #309](https://github.com/3scale/apicast/issues/309)
```

Comment:
I think this can stay in the release notes. Because it was a known issue for that version.